local_settings.py.tmpl

```diff
@@ -257,6 +253,27 @@ MAP_BASELAYERS = [
     }
 ]
 
+# Uncomment to allow open registration.
+# FIXME: there's no admin validation on new accounts.
+REGISTRATION_OPEN = True
```

Comment:
Should this boolean variable be kept on ?

Reply:
In Geonode settings.py is of default: False